### ISSUE TITLE
Use consistent names for :group-id/:artifact-id

### ISSUE THREAD

**Issue by tobias** [clojars/clojars-web#525]
Depending on where you are in the codebase, we use `:group`/`:name`, `:group-id`/`:artifact-id`, `:group_name`/`:jar_name`, and possibly others. This means we end up doing a lot of key renaming when crossing boundaries (db/index/poms), which is error-prone and mentally inefficient.

**Comment by danielcompton** [comment]
Good plan. We had the same problem at work recently and created a glossary document defining standard names and their definitions.

**Comment by mynomoto** [comment]
Is `:group-id/:artifact-id` the canonical one?

**Comment by danielcompton** [comment]
I think that should probably be?

**Comment by tobias** [comment]
Yes, those are the preferred names. Be aware that this is a large change, and may be painful to implement :)
